app/src/main/java/ca/bc/gov/vaxcheck/ui/onboarding/OnBoardingFragment.kt

```diff
@@ -61,7 +64,13 @@ class OnBoardingFragment : Fragment(R.layout.fragment_onboarding) {
         }
 
         binding.txtPrivacyPolicy.setSpannableLink {
-            findNavController().navigate(R.id.action_onBoardingFragment_to_webViewFragment)
+            val webpage: Uri = Uri.parse(getString(R.string.url_privacy_policy))
+            val intent = Intent(Intent.ACTION_VIEW, webpage)
+            try {
+                startActivity(intent)
+            } catch (e: Exception) {
+                Toast.makeText(context, getString(R.string.no_app_found), Toast.LENGTH_LONG).show()
+            }
```

Comment:
I would like to suggest:
- [ ]  creating a specific method for this, since `onViewCreated` is getting big

Reply:
separate function created.